### PR TITLE
List of players on game history

### PR DESCRIPTION
**Pull request by vitsalis**

### CODE REVIEW

**Comment by vitsalis** [comment]
selection_001
selection_002

**Comment by dionyziz** [comment]
Please illustrate the visuals of the 'you' CSS class with a screenshot. Does it work?

File: views/game/view.php [hunk 1]
```diff
@@ -10,6 +10,44 @@
...
 
+<?php
+    $colors = [ 'red', 'blue', 'green', 'yellow', 'purple', 'cyan', 'black', 'pink' ];
+    $playerColor = [];
+    foreach ( $game->users as $user ) {
+        $playerColor[ $user->id ] = array_shift( $colors );
+    }
+    foreach ( $round->creatures as $creature ) {
+        if ( $creature->alive ) {
+            if ( !isset( $hasCreatures[ $creature->user->id ] ) ) {
```
Comment (dionyziz) [inline views/game/view.php:22]: initialize `$hasCreatures`

**Comment by vitsalis** [comment]
selection_003

**Pull request merged**